app/services/katello/pulp3/migration.rb

```diff
@@ -114,6 +114,40 @@ def migration_plan
         Katello::Pulp3::MigrationPlan.new(@repository_types).generate.as_json
       end
 
+      def reset
+        if @repository_types.empty?
+          fail ::Katello::Errors::Pulp3MigrationError, 'There are no Pulp 3 content types to reset'
+        end
+
+        plugins = @repository_types.sort.map do |repository_type|
+          {
+            type: ::Katello::Pulp3::MigrationPlan.pulp2_repository_type(repository_type)
+          }
+        end
+        plan = { plugins: plugins }
+
+        # TODO: Don't provide the plan as a string once this is resolved: https://pulp.plan.io/issues/8211
+        migration_plan_api.reset(migration_plan_api.create(plan: plan).pulp_href, plan.to_json)
+
+        content_types_for_migration.each do |content_type|
+          content_type.model_class.all.update(migrated_pulp3_href: nil)
```

Comment:
i think you want to use update_all here to not go through AR and execute a single update